tests/testthat/test-excel_time_to_numeric.R

```diff
@@ -0,0 +1,162 @@
+context("Excel time cleaning")
```

Comment:
I still need to try out the tests and review them but don't want that to hold up my feedback on the actual code.